.github/workflows/bindings-wasm-publish.yml

```diff
@@ -49,3 +49,18 @@ jobs:
         env:
           NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
         run: npm publish --access public
+
+      - name: Prepare Github release
+        id: prepare_release
+        run: |
+          yarn pack
+          echo "tag_name=iota-sdk-wasm-v$(npm pkg get version --workspaces=false | tr -d \")" >> "$GITHUB_OUTPUT"
+
+      - name: Upload package to Github release
+        uses: softprops/action-gh-release@v1
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          body: https://github.com/iotaledger/iota-sdk/blob/develop/bindings/wasm/CHANGELOG.md \n https://github.com/iotaledger/iota-sdk/blob/develop/bindings/nodejs/CHANGELOG.md
+          files: bindings/wasm/*.tgz
+          tag_name: ${{ steps.prepare_release.outputs.tag_name }}
+          prerelease: true
```

Comment:
what are our conditions to no longer be pre-release?

Reply:
I guess we'll have to remove it after we release 1.0